wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -208,28 +210,27 @@ def _add_to_launch_queue(
     ) -> "public.QueuedRun":
         """Add a launch job to the Launch RunQueue."""
         run_id = run_id or generate_id()
+        # One of Job and URI is required
+        _job = self._kwargs.get("job", None)
+        _uri = self._kwargs.get("uri", None)
+        if _job is None and _uri is None:
+            # If no Job is specified, use a placeholder URI to prevent Launch failure
```

Comment:
@KyleGoyette I wonder if there is a better way of handling this on the Launch side, I also have had to write temp URI's which kinda defeats the purpose of the check.

Reply:
Yeah, I guess this pattern is used when we just want to run command in a process that doesn't refer to a run or file? This is fine for now, but we should potentially think about that. Maybe just check if there's an entrypoint?